src/ui/public/vislib/lib/dispatch.js

```diff
@@ -214,33 +215,33 @@ export default function DispatchClass(Private) {
      * Mouseover Behavior
      *
      * @method addMousePointer
-     * @returns {D3.Selection}
+     * @returns {d3.Selection}
```

Comment:
the D3->d3 rename is to silence the jsdoc linter, who does not recognize D3 as a valid type.